data/blog/mike-dilger-receives-lts-grant.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: "Long-Term Support For Mike Dilger"
+date: '2024-09-15'
+tags: ['OpenSats', 'grants', 'nostr', 'LTS']
+draft: false
+authors: ['arvin', 'dergigi']
+images: ['/static/images/blog/52-mike-dilger-lts.jpg']
+summary: "OpenSats Welcomes Mike Dilger as an LTS Grantee."
+---
+
+OpenSats is excited to announce long-term support for Mike Dilger, a dedicated
+contributor in the nostr ecosystem.
+
+Mike is known for being the lead developer of
+[Gossip](https://github.com/mikedilger/gossip?tab=readme-ov-file#), a nostr
+client designed to prioritize privacy and security by avoiding [common web stack
+vulnerabilities](https://owasp.org/www-project-top-ten/#) and leveraging rust
+language security.
+
+Gossip has become a reference implementation for the "[outbox
+model](https://mikedilger.com/gossip-model/)", a design that enables more
+scalable message distribution across the nostr network. In addition to Gossip,
+Mike has contributed to several tools and libraries that support the nostr
+ecosystem, such as the [chorus relay](https://github.com/mikedilger/chorus),
+[relay tester](https://github.com/mikedilger/relay-tester), and his active
+[participation in NIP
+discussions](https://github.com/nostr-protocol/nips/issues?q=involves%3Amikedilger).
+
+With the support of this LTS grant, Mike plans to continue his work on the
+Gossip client, with a focus on implementing key features such as:
+
+- Improving support for replying to various versions of replaceable events.
+
+- Adding [NIP-96](https://github.com/nostr-protocol/nips/blob/master/96.md) file
+  uploads.
+
+- Implementing Blossom support, Web of Trust, and search at relay features.
```

Comment:
> search at relay features.

I'm not sure what that means?

Reply:
I was not sure initially when writing the first draft. It was part of the original source material. But in reading it again before opening the PR, I was fairly certain that it means having search capabilities at the relay level, to be able to search events from a specific relay. Perhaps I should reword this to "...and relay search functionality."